packages/manager/apps/vrack-services/src/pages/dashboard/[id]/components/SubnetDataGridCells.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { UseMutateAsyncFunction } from '@tanstack/react-query';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+import {
+  ODS_BUTTON_SIZE,
+  ODS_BUTTON_TYPE,
+  ODS_BUTTON_VARIANT,
+} from '@ovhcloud/ods-components/button';
+import { OsdsButton } from '@ovhcloud/ods-components/button/react';
+import { OsdsIcon } from '@ovhcloud/ods-components/icon/react';
+import { ODS_ICON_NAME, ODS_ICON_SIZE } from '@ovhcloud/ods-components/icon';
+import { OsdsClipboard } from '@ovhcloud/ods-components/clipboard/react';
+import { EditableText } from '@/components/EditableText';
+import {
+  ResponseData,
+  Subnet,
+  UpdateVrackServicesParams,
+  VrackServices,
+} from '@/api';
+import { DataGridCellProps, handleClick } from '@/utils/ods-utils';
+import { ApiError } from '@/components/Error';
+import { isEditable } from '@/utils/vs-utils';
+
+type UpdateVS = UseMutateAsyncFunction<
+  ResponseData<VrackServices>,
+  ResponseData<ApiError>,
+  UpdateVrackServicesParams
+>;
+
+export const DisplayNameCell: React.FC<DataGridCellProps<
+  string | undefined,
+  Subnet
+> & {
+  vrackServices?: VrackServices;
+  updateVS: UpdateVS;
+}> = ({ cellData, rowData, updateVS, vrackServices }) => (
+  <EditableText
+    disabled={!isEditable(vrackServices)}
+    defaultValue={cellData}
+    onEditSubmitted={async (value) => {
+      await updateVS({
+        vrackServicesId: vrackServices.id,
+        checksum: vrackServices.checksum,
+        targetSpec: {
+          displayName: vrackServices.currentState.displayName || null,
+          subnets: vrackServices.currentState.subnets.map((subnet) =>
+            subnet.cidr === rowData.cidr
+              ? {
+                  ...subnet,
+                  displayName: value,
+                }
+              : subnet,
+          ),
+        },
+      });
+    }}
+  >
+    {cellData}
+  </EditableText>
+);
+
+export const CidrCell: React.FC<DataGridCellProps<
+  string | undefined,
+  Subnet
+>> = ({ cellData }) => <OsdsClipboard value={cellData} inline />;
+
+export const ActionsCell: React.FC<DataGridCellProps<undefined, Subnet> & {
+  isLoading?: boolean;
+  updateVS: UpdateVS;
+  vrackServices?: VrackServices;
+}> = ({ vrackServices, rowData, isLoading, updateVS }) => {
+  const editable = isEditable(vrackServices);
+
+  return (
+    <OsdsButton
+      inline
+      color={ODS_THEME_COLOR_INTENT.error}
+      variant={ODS_BUTTON_VARIANT.stroked}
+      type={ODS_BUTTON_TYPE.button}
+      size={ODS_BUTTON_SIZE.sm}
+      disabled={isLoading || !editable || undefined}
+      {...handleClick(() =>
```

Comment:
Why not pass props in classic way 
```
handleClick={}
```

Reply:
it's an helper to add the keyboard handler as well as the onClick handler for keyboard accessibility